src/wireviz/wv_helper.py

```diff
@@ -152,6 +152,9 @@ def graphviz_line_breaks(inp):
 def remove_line_breaks(inp):
     return inp.replace('\n', ' ').strip() if isinstance(inp, str) else inp
```

Comment:
I suggest removing the no longer used functions `graphviz_line_breaks()` and `remove_line_breaks()` above.